pandas/io/parsers/readers.py

```diff
@@ -268,7 +268,17 @@ class _read_shared(TypedDict, Generic[HashableT], total=False):
 skipfooter : int, default 0
     Number of lines at bottom of file to skip (Unsupported with ``engine='c'``).
 nrows : int, optional
-    Number of rows of file to read. Useful for reading pieces of large files.
+    Number of rows of data to read. Useful for reading pieces of large files.
+    The following rows are excluded from the count:
+
+    * The header row containing column names,
+    * Rows before the header row, if ``header=1`` or larger,
+    * Fully commented rows,
+    * Rows skipped with ``skiprows``,
+    * Skipped blank lines.
+    
+    Records containing text with newline characters within quotes
+    or escaped newline characters will be counted as one row.
```

Comment:
This is related to how `read_csv` parses the file and not just related to the `nrows` parameter, I think this should also be removed.

Reply:
Removed